mage_ai/frontend/pages/pipelines/[pipeline]/triggers/index.tsx

```diff
@@ -197,25 +197,39 @@ function PipelineSchedules({
   const buildSidekick = useMemo(() => {
     const variablesOverride = selectedSchedule?.variables;
     const hasOverride = !isEmptyObject(variablesOverride);
-
-    const showVariables = hasOverride
-      ? selectedSchedule?.variables
-      : !isEmptyObject(variablesOrig) ? variablesOrig : null;
+    const hasVariables = !isEmptyObject(variablesOrig);
```

Comment:
The code in `RuntimeVariables` indicates that a variable override is only relevant if the variable exists in `variablesOrig`, so I simplified the logic here to only check if original variables are set.